tests/src/Functional/ContentRenderTestBase.php

```diff
@@ -334,6 +336,39 @@ protected function createVenueEntity(string $name, string $bundle = 'oe_default'
     return $venue;
   }
 
+  /**
+   * Creates Programme item entity based on provided settings.
+   *
+   * @param string $name
+   *   Entity name. Is used as a parameter for test data.
+   * @param string $bundle
+   *   Entity bundle.
+   * @param int $status
+   *   Entity status.
+   *
+   * @return \Drupal\oe_content_event_event_programme\Entity\ProgrammeItemInterface
+   *   Venue entity instance.
+   */
+  protected function createProgrammeItemEntity(string $name, string $bundle = 'oe_default', int $status = CorporateEntityInterface::PUBLISHED): ProgrammeItemInterface {
```

Comment:
Would it be useful to have this in a corporate entity trait somewhere? Might not, just an idea.

Reply:
This method looks similar as other methods like `createVenueEntity`, `createMediaDocument`, `createMediaImage` etc. For now, I didn't find big demand for suck reorganising of code.